We do not want the Python version in test results because otherwise we would
need to normalize it out anyway.

--- waitress-2.1.2/tox.ini.orig
+++ waitress-2.1.2/tox.ini
@@ -8,7 +8,6 @@
 
 [testenv]
 commands =
-    python --version
     python -mpytest \
     pypy38: --no-cov \
         {posargs:}
